src/main/java/org/damap/base/rest/AdminResource.java

```diff
@@ -51,4 +63,82 @@ public void deleteAppBanner() {
     log.info("DELETE /admin/banner");
     this.adminService.deleteAppBanner();
   }
+
+  @GET
+  @Path("/color-theme")
+  @RolesAllowed("Damap Admin")
+  public ColorThemeDO getColorTheme() {
+    log.info("GET /admin/color-theme");
+    return this.colorThemeService.getTheme();
+  }
+
+  @PUT
+  @Path("/color-theme")
+  @RolesAllowed("Damap Admin")
+  public ColorThemeDO uploadColorTheme(@Valid ColorThemeDO colorThemeDO) {
+    log.info("PUT /admin/color-theme");
+    return colorThemeService.uploadTheme(colorThemeDO);
+  }
+
+  @PUT
+  @Path("/image-theme")
+  @RolesAllowed("Damap Admin")
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  public ImageDO uploadImage(
+      @FormParam("imageKey") String imageKey, @FormParam("file") FileUpload file) {
```

Comment:
I am vey surprised that this works, since `@FormParam` is supposed to be used for [rest clients](https://quarkus.io/guides/rest-client#form-parameters) 😅
Intellij even complains about that:
<img width="263" height="36" alt="image" src="https://github.com/user-attachments/assets/c8c41bd1-c24d-4e7d-a25f-b05a2740d50a" />

Here are the [docs](https://quarkus.io/guides/rest#multipart) on how to do the server end.

Reply:
As discussed, I think we should follow the docs (both work, but its saver to stick to them) 
```suggestion
  @Consumes(MediaType.MULTIPART_FORM_DATA)
  public ImageDO uploadImage(
      @RestForm("imageKey") String imageKey, @RestForm("image") FileUpload file) {
```

Reply:
Done